13LessonThirteen/in-class-project/css/style.css

```diff
@@ -0,0 +1,76 @@
+img {
+    width: 100%;
+}
+
+main {
+    padding: 15px;
+}
+
+.button, footer {
+    color: white;
+    background-color: #5d5d5d;
+}
+
+.button {
+    padding: 6px;
+    text-decoration: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+h1 {
+    font-size: 32px;
+    font-weight: bold;
+}
+
+p {
+    font-size: 24px;
+    font-weight: bold;
+}
+
+aside h2 {
+    font-weight: bold;
+    text-transform: uppercase;
```

Comment:
It's up for debate whether if it would be better to rely on using `text-transform` to capitalize the first letter of all these words, or to just capitalize the words in the `aside` element itself. I personally would opt for the latter - let's say a few weeks pass and you need to make some revisions to your files. By that time you might have forgotten that you used the `text-transform` property on this element, and it might create an unnecessary source of confusion when you're trying to change the text capitalization back to normal. Whereas if you just format the text that way in the element itself, you can quickly see that by simply looking at it in your HTML.